.Rbuildignore

```diff
@@ -30,4 +30,5 @@ CONDUCT.md
 ^revdep$
 ^inst/ciebg\.svg$
 ^inst/linear_visible_spectrum\.svg$
+^vignettes/fig/workflow\.png$
```

Comment:
Maybe instead of doing this, I should put this image again in `pavo-1-overview.Rmd`? It seems to have been lost in 58add92ae897dd36732b95346a1b993be07de233

Reply:
Yeah right, I wonder if I did that on purpose.

It's a useful figure anyway, so I'd be included to pop it back into `overview` if it's not a burden on size etc. Otherwise everything looks great — cheers.